frontend/src/components/housing/submission/SubmissionIntakeForm.vue

```diff
@@ -492,6 +500,12 @@ onBeforeMount(async () => {
     }
 
     locationRef.value?.onLatLongInput();
+    // Load BCeID business name
+    bceidBusinessName.value = (
```

Comment:
Shouldn't the BCeID business name already be loaded into the store from oidc callback view?